packages/cactus-plugin-keychain-memory/src/main/typescript/get-prometheus-exporter-metrics/get-prometheus-exporter-metrics-endpoint-v1.ts

```diff
@@ -0,0 +1,81 @@
+import { Express, Request, Response } from "express";
+
+import {
+  Logger,
+  LoggerProvider,
+  LogLevelDesc,
+  Checks,
+} from "@hyperledger/cactus-common";
+
+import {
+  IWebServiceEndpoint,
+  IExpressRequestHandler,
+} from "@hyperledger/cactus-core-api/";
+
+import OAS from "../../json/openapi.json";
+
+import { registerWebServiceEndpoint } from "@hyperledger/cactus-core";
+
+import { PluginKeychainMemory } from "../plugin-keychain-memory";
+
+export interface IGetPrometheusExporterMetricsEndpointV1Options {
+  logLevel?: LogLevelDesc;
+  connector: PluginKeychainMemory;
```

Comment:
@jagpreetsinghsasan I'd call it something else, not a connector. Either would call it maybe `keychain` or just go with `plugin` which makes it more copy-paste-proof for the future ;-)